Package.swift

```diff
@@ -356,12 +378,14 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "0.4.3")),
         .package(url: "https://github.com/apple/swift-driver.git", .branch(relatedDependenciesBranch)),
         .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: "1.1.4")),
+        .package(url: "https://github.com/apple/swift-syntax.git", .branch(relatedDependenciesBranch)),
```

Comment:
In principle this is correct, but SwiftSyntax is particularly tricky, because of its dependency on the corresponding linker library.  In fact I wasn't able to get the `main` branch to work with Xcode 13 Betas at all.  But as it happens we have two other intended uses of SwiftSyntax, and so I will be working with someone who knows it better than I do to add it as a dependency in a separate PR.  So you can keep this as it is right now, and within not too long we should be able to have this dependency already added through a separate PR.

Reply:
That's weird because I'm building it with Xcode 13 Beta 4/5, on Big Sur.
I think the dependency could be dropped here some day for parsing, but it would help doing things like generating a package.